File: csharp/beginner/004_Conditionals.cs
// [4] 조건문 (if-switch) - Conditionals & Pattern Matching
// 레벨: 2 | C#의 조건문과 패턴 매칭을 배웁니다

using System;

class Program
{
    static void Main()
    {
        int score = 75;
        Console.WriteLine(score >= 80 ? "high" : "low");

        int day = 1;
        Console.WriteLine(day switch
        {
            1 => "Mon",
            _ => "Other"
        });

        object v = 3;
        if (v is int n) Console.WriteLine(n);

        int age = 6;
        Console.WriteLine(age < 8 ? 0 : 100);
    }
}

File: java/beginner/004_Conditionals.java
// [4] 조건문 (if-switch) - Conditionals
// 레벨: 2 | Java의 조건문을 사용해 프로그램 흐름을 제어합니다

public class Conditionals {

    public static void main(String[] args) {

        int score = 85;
        String grade;

        if (score >= 90) grade = "A";
        else if (score >= 80) grade = "B";
        else grade = "C";

        System.out.println("Grade " + grade);

        int day = 3;
        String dayName;

        switch (day) {
            case 1: dayName = "Mon"; break;
            case 2: dayName = "Tue"; break;
            case 3: dayName = "Wed"; break;
            default: dayName = "Other";
        }

        System.out.println("Day " + dayName);

        int age = 20;
        String status = (age >= 18) ? "Adult" : "Minor";
        System.out.println(status);
    }
}
